Automattic/wp-calypso
# Membership tiers

# OLD - Rebased in https://github.com/Automattic/wp-calypso/pull/82728




Fixes to https://github.com/Automattic/gold/issues/140
Related to https://github.com/Automattic/jetpack/pull/32658
And D119740-code



## Proposed Changes

* Change plan list according to Figma
* Add Newsletter tier to bundle 2 plans
* Change the add/edit modal to handle such changes
* Lock the Newsletter selector when a plan was added as-is.
* It should be retro-compatible with existing plans.
* Display only one success notice for tiers
* Fix: only show "you can now add your first payment plan" if needed  (cannot be tested as Stripe sends you back to WP.com)

| list | modal |
| ---- | ---- |
|  <img width="1146" alt="Screenshot 2023-08-24 at 19 44 08" src="https://github.com/Automattic/wp-calypso/assets/790558/81e4d779-3c98-4877-9383-60adb2cd29a6"> |  <img width="769" alt="Screenshot 2023-08-24 at 19 44 17" src="https://github.com/Automattic/wp-calypso/assets/790558/0dcd5420-2a6d-4e97-878b-b085ddfd1c75"> | 



## Testing Instructions



* Patch D119740-code on your sandbox
* Add new plans, delete and modify to test
* Have a site with a monthly newsletter plan and a yearly newsletter plan. Try to edit them with this patch on. It should created bundled plans.

CC @crisbusquets for designs recitifications
